app/routes/index.js

```diff
@@ -86,7 +86,20 @@ router.get('/getBlogPosts', (req, res) => {
             .then(response => {
               const data = response.getBody();
               data.posts = data.posts.filter(post => {
-                return post.state == 'published';
+                console.log(
```

Comment:
I don't think we need the `console.log()` in the index.js?

Reply:
Oops forgot to get rid of that lmao